<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>251</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>math option</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>481</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>옵션</string>
   </property>
   <widget class="QRadioButton" name="mean_radio">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>108</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>평균</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="sum_radio">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>108</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>합계</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="cal_radio">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>계산</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="complete_btn">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>180</y>
     <width>93</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>완료</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
